alcs-frontend/src/app/features/application/info-requests/info-rquest-dialog/info-request-dialog.component.html

```diff
@@ -19,7 +19,6 @@ <h2 class="card-title">{{ model.uuid ? 'Update' : 'Create' }} Information Reques
           />
           <mat-datepicker-toggle matSuffix [for]="startDate"></mat-datepicker-toggle>
           <mat-datepicker #startDate type="date"> </mat-datepicker>
-          <mat-hint>{{ model.startDate | date: 'mediumDate' }}</mat-hint>
```

Comment:
No hints anymore? 🥲

Reply:
Confirmed with UX we will be removing hints